src/cairo_helpers.lua

```diff
@@ -157,12 +157,27 @@ local function text_extents(cr, text)
     return extents
 end
 
+--- Round coordinates to even pixel values
+--- https://www.cairographics.org/FAQ/#sharp_lines
```

Comment:
Three dashes only on the first line as per [LDoc](https://stevedonovan.github.io/ldoc/manual/doc.md.html). Also use @see.